src/crypto/CHIPCryptoPALPSA.cpp

```diff
@@ -40,6 +40,7 @@
 #include <mbedtls/oid.h>
 #include <mbedtls/x509.h>
 #include <mbedtls/x509_csr.h>
+#include <mbedtls/threading.h>
```

Comment:
Why is this required now ?

Reply:
It wasn't building previously without this, I'll try it without again and see if it is still required.